enterprise/cypher/acceptance-spec-suite/src/test/scala/org/neo4j/internal/cypher/acceptance/SpatialFunctionsAcceptanceTest.scala

```diff
@@ -30,6 +30,11 @@ class SpatialFunctionsAcceptanceTest extends ExecutionEngineFunSuite with Cypher
   val equalityConfig = Configs.Interpreted - Configs.OldAndRule
   val latestPointConfig = Configs.Interpreted - Configs.BackwardsCompatibility - Configs.AllRulePlanners
 
+  test("toString on points") {
+    val result = executeWith(latestPointConfig,  "RETURN toString(point({x:1, y:2})) AS s")
+    result.toList should equal(List(Map("s" -> "Point{cartesian, [1.0, 2.0]}")))
```

Comment:
We would like the toString version of Point to be something standard. For cypher-shell the decision was to 'PrettyPrint' to the same format that Cypher could parse. Since BOLT only transmitted the CRS code, not CRS name, we actually ended up adding support for 'SRID' key to the Cypher point() function just to support this. I think we need to do the same here.

Reply:
Originally we decided not to support toString because the plan was to use WKT, but that has several problems, one is that WKT does not obviously support the CRS. Now that we have a separate decision for the cypher-shell, it seems sensible to follow that for consistency.

Reply:
Clearly Cypher-shell has not yet fixed their problem. The decision a few weeks back was they would show `point({x: 2.0, y: 3.0, srid: 7203})`. The use of srid was only because they did not have the CRS name. So we could print either `point({x: 2.0, y: 3.0, srid: 7203})` or `point({x: 2.0, y: 3.0, crs: 'cartesian'})`, which are now equivalent.